modules/va_facilities/app/controllers/va_facilities/v0/facilities_controller.rb

```diff
@@ -79,8 +79,9 @@ def validate_bbox_lat_and_long_or_ids
         bbox = params.key?(:bbox)
         lat_and_long = params.key?(:lat) && params.key?(:long)
         ids = params.key? :ids
+        state = params.key? :state
 
-        if !bbox && !lat_and_long && !ids
+        if !bbox && !lat_and_long && !ids && !state
```

Comment:
I don't think `!state` belongs here.  Or maybe it does and this method needs to be renamed.  But right now it looks like it's raising an exception based on what it's missing from `%i[bbox long lat ids]`.  `:state` isn't on that list.  If it should be I think this method ought to have a more descriptive name of what it's doing, or state should have it's own validation method.

Reply:
This is the function that was causing the cyclomatic complexity lint error for me last week. I agree that the name isn't very good. I changed it to `validate_a_param_exists`, because the point is to make sure one of these params exists, but if anyone has a better idea . . .

Reply:
Agree that it's getting quite complex with the number of params it's requiring. I'm planning to steal from @nfasulo's change.

Reply:
Followed the link in slack and I think Nick's change is looking good